build-windows.bat

```diff
@@ -0,0 +1,17 @@
+@echo off
```

Comment:
Should this file be under python/? Or alternatively, call it build-windows-experimental.bat?

(This does not seem like it will build the .NET bindings, and that is one of the major Windows targets)

Reply:
build-windows-experimental.bat doesn't actually build the python bindings, just the main library (with associated dll/exes)